package.json

```diff
@@ -26,7 +26,8 @@
 		"README.md"
 	],
 	"scripts": {
-		"test": "node --experimental-vm-modules node_modules/.bin/jest"
+		"test": "node --experimental-vm-modules node_modules/.bin/jest",
+		"postinstall": "npx simple-git-hooks"
```

Comment:
Let's just copy over the [`.editorconfig`](https://github.com/preactjs/wmr/blob/3c5672ecd2f958c8eaf372d33c084dc69228ae3f/.editorconfig), I don't think we need precommit hooks here.

Reply:
Ah, sorry, just curious, why use two different types of configuration files like prettier/.editorconfig? Additionally, postInstall is for automatically adding a git hook, which is no different from the husky configuration in wmr/preact-router, and of course, the automatic hook can also be removed.

Reply:
> why use two different types of configuration files like prettier/.editorconfig?

Slightly different tools with different uses/responsibilities.

> Additionally, postInstall is for automatically adding a git hook, which is no different from the husky configuration in wmr/preact-router, and of course, the automatic hook can also be removed.

I'm aware. I wouldn't call it useful in either of those repos either.

> Ah, just found out that the [.editorconfig](https://github.com/preactjs/preact-iso/blob/main/.editorconfig) configuration file already exists.

Whoops. This should address the formatting then as-is.